docs/template_extension/cpu_kernel.cpp

```diff
@@ -23,6 +23,8 @@ OpImplementation::OpImplementation(const std::shared_ptr<ngraph::Node> &node) {
         if (castedNode->get_input_element_type(0) != ngraph::element::f32 || castedNode->get_output_element_type(0) != ngraph::element::f32)
             THROW_IE_EXCEPTION << "Operation supports only FP32 tensors.";
         add = castedNode->getAddAttr();
+        inShape = castedNode->get_input_shape(0);
```

Comment:
What if there are more that one input?

Reply:
We have a check in this line: https://github.com/openvinotoolkit/openvino/pull/1695/files#diff-7b34831df09cb077dde39d1b3c043a46R17

Reply:
@dkurt Is this PR work for your case?

Reply:
```
terminate called after throwing an instance of 'InferenceEngine::details::InferenceEngineException'
  what():  Cannot fill descriptor. Size of dimensions and order vector don't match.
Aborted (core dumped)
```
for now. Will try to check the total diff comparing to master

Reply:
Ok, It happens because we hardcoded the order. I will modify the code.

Reply:
Actually, this appears only after network's reshape. Origin shapes inference works fine, thanks!